startup_runners/startup_runner.c

```diff
@@ -13,19 +13,20 @@
 #include <errno.h>
 #include <dlfcn.h>
 #include <err.h>
+#include <stdint.h>
 #include <unistd.h>
 
-#define BENCH_FUNC_NAME "run_iter"
+#include "libkruntime.h"
 
-/* from libkruntime */
-double clock_gettime_monotonic();
+#define BENCH_FUNC_NAME "run_iter"
 
 int
 main(void)
 {
     double    start_time = -1;
 
-    start_time = clock_gettime_monotonic();
+    krun_measure(0);
```

Comment:
Ah, I see what you're saying. I don't think there's a problem with using index 1.

`krun_measure` fills out `&(krun_mdata[mdata_idx])` and `krun_get_wallclock` reads from that. The index check just seems to check if the index is valid, not whether data has ever been read/written to that slots.